src/app/tile-header/tile-header.component.html

```diff
@@ -0,0 +1,5 @@
+<kendo-tilelayout-item-header>
+  <div>{{title}}</div>
+  <ng-content>
+  </ng-content>
```

Comment:
Add two named contents so that we can choose where we display children: 
https://angular.io/guide/content-projection#multi-slot-content-projection

This one is all right, just add one that will allow to display children inside a menu showed by:
https://www.telerik.com/kendo-angular-ui/components/buttons/dropdownbutton/